http/graphql/src/test/java/io/quarkus/ts/http/graphql/TracingIT.java

```diff
@@ -0,0 +1,70 @@
+package io.quarkus.ts.http.graphql;
+
+import static io.quarkus.ts.http.graphql.Utils.createQuery;
+import static io.quarkus.ts.http.graphql.Utils.sendQuery;
+import static io.restassured.RestAssured.when;
+import static org.awaitility.Awaitility.await;
+
+import java.net.HttpURLConnection;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.JaegerService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.JaegerContainer;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+public class TracingIT {
+    private static final String SERVICE_NAME = "graphql-service";
+
+    @JaegerContainer
+    static JaegerService jaeger = new JaegerService();
+
+    @QuarkusApplication
+    static RestService app = new RestService()
+            .withProperty("quarkus.jaeger.service-name", SERVICE_NAME)
+            .withProperty("quarkus.jaeger.endpoint", jaeger::getRestUrl);
+
+    @Order(1)
```

Comment:
Tests should (ideally) be not ordered and able to be ran independently on their execution